shared/src/packages/trucks/libs/validation-schemas/truck-id-parameter.validation-schema.ts

```diff
@@ -0,0 +1,11 @@
+import joi from 'joi';
+
+import { TruckValidationMessage } from '../enums/truck-validation-message.enum.js';
+
+const truckIdParameter = joi.object({
+  id: joi.number().integer().positive().required().messages({
+    'number': TruckValidationMessage.ID_NOT_A_NUMBER,
+  }),
+});
```

Comment:
in development there is schema ( it was added by @MaxHopaytsa I think, you can connect with him in cases of any questions) for validating positive id number, reuse it